distro/pom.xml

```diff
@@ -640,6 +640,40 @@
           </execution>
 
           <execution>
+                <id>Copy OpenMRS SPA configuration</id>
```

Comment:
Should we add these in maven-parent as well?

Reply:
In that case `Copy and rename frontend configuration` should be added to `maven-parent`